x/group/README.md

```diff
@@ -578,6 +579,15 @@ The group module emits the following events:
 | cosmos.group.v1.EventLeaveGroup | proposal_id   | {proposalId}                    |
 | cosmos.group.v1.EventLeaveGroup | address       | {address}                       |
 
+### EventProposalFinalized
+
+| Type                                   | Attribute Key | Attribute Value                 |
+|----------------------------------------|---------------| ------------------------------- |
+| message                                | action        | /cosmos.group.v1.Msg/LeaveGroup |
+| cosmos.group.v1.EventProposalFinalized | proposal_id   | {proposalId}                    |
+| cosmos.group.v1.EventProposalFinalized | status        | {ProposalStatus}                       |
```

Comment:
nit: can you align those table ending :D
markdown all-in-one is vscode can do that, and then you can markdownlint (if it changed anything else), or manually may be faster 😅